src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -826,9 +826,9 @@ private void TouchEverything(Solution solution)
             {
                 foreach (var document in project.Documents)
                 {
-                    document.GetTextAsync().PumpingWait();
-                    document.GetSyntaxRootAsync().PumpingWait();
-                    document.GetSemanticModelAsync().PumpingWait();
+                    document.GetTextAsync().Wait();
+                    document.GetSyntaxRootAsync().Wait();
+                    document.GetSemanticModelAsync().Wait();
```

Comment:
I cannot say. Although the workspace tasks in the background will always schedule all their work to the thread pool, I could imagine this is trying to hook into some event stream and possibly trying to react. @heejaechang might know better.